### PR TITLE
Add output fields value check

### PR DESCRIPTION
**Pull request by NicoYuan1986**

### CODE REVIEW

File: tests/python_client/testcases/test_query.py [hunk 1]
```diff
@@ -600,8 +600,7 @@ def test_query_output_float_vec_field(self):
 
     @pytest.mark.tags(CaseLabel.L2)
     @pytest.mark.parametrize("wildcard_output_fields", [["*"], ["*", default_float_field_name],
-                                                        ["*", default_int_field_name],
-                                                        ["%"], ["%", default_float_field_name], ["*", "%"]])
```
Comment (binbinlv) [inline tests/python_client/testcases/test_query.py:604]: what is the behavior of "%", any case cover this behavior?

Comment (NicoYuan1986) [reply tests/python_client/testcases/test_query.py:604]: > what is the behavior of "%", any case cover this behavior?

'%' is invalid now with message `field % not exist`

Comment (NicoYuan1986) [reply tests/python_client/testcases/test_query.py:604]: I have added the check cases of invalid output fields

File: tests/python_client/testcases/test_search.py [hunk 2]
```diff
@@ -1810,7 +1808,7 @@ def test_search_max_nq(self, auto_id, dim, _async):
         expected: search successfully with max nq
         """
         self._connect()
-        nq = 17000
+        nq = 16384
```
Comment (binbinlv) [inline tests/python_client/testcases/test_search.py:1811]: Do we have case for nq=16385? (maximum+1)?

Comment (NicoYuan1986) [reply tests/python_client/testcases/test_search.py:1811]: > Do we have case for nq=16385? (maximum+1)?

No.

Comment (NicoYuan1986) [reply tests/python_client/testcases/test_search.py:1811]: > Do we have case for nq=16385? (maximum+1)?

Added in #25317.

**Pull request merged**